src/main/webapp/index.html

```diff
@@ -2,18 +2,46 @@
 <html>
   <head>
     <meta charset="UTF-8">
-    <title>My Portfolio</title>
+    <title>Project</title>
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
   </head>
   <body>
-    <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click <a href="/hello">here</a> to see content created by server code.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+    <div id="login">
+        <script src="https://accounts.google.com/gsi/client" async defer></script>
+        <script src="https://unpkg.com/jwt-decode/build/jwt-decode.js"></script>
+        <script>
+          function handleCredentialResponse(response) {
+              const data = jwt_decode(response.credential)
+              console.log(data)
+              fullName.textContent = data.name
+              sub.textContent = data.sub
+              given_name.textContent = data.given_name
+              family_name.textContent = data.family_name
+              email.textContent = data.email
+              verifiedEmail.textContent = data.email_verified
+              picture.setAttribute("src", data.picture)
```

Comment:
For these, I think you need to use document.getElementById, here are some examples: https://developer.mozilla.org/en-US/docs/Web/API/Node/textContent#examples

Reply:
Please see my latest code.